src/mono/mono/eglib/eglib-remap.h

```diff
@@ -29,11 +29,6 @@
 #define g_clear_error monoeg_g_clear_error
```

Comment:
embedders can still potentially static link with us and the "real" glib.  in that case they might have a symbol clash, right?

Reply:
OK, but I assume that we don't support embedders linking against runtime and assume it has glib symbols implemented. Reason why I ask is because we change and drop g_* functions from our eglib implementation, so I assume our eglib is private only and for runtime consumption. Eliminate symbol collision with with static linked glib still make sense. Maybe we should move away from using g_* symbols inside runtime and eliminate the need for the remapping.